airflow/executors/celery_executor.py

```diff
@@ -256,6 +260,12 @@ def sync(self):
         self._sync_pool.join()
         self.log.debug("Inquiries completed.")
 
+        self.update_task_states(task_keys_to_states)
+
+    def update_task_states(self,
```

Comment:
I think this function is not required as we don't use it elsewhere unless of course I have not seen it yet :). `update_task_state` is fine.

Reply:
But it's actually used there - List of tasks from map :).. So it is used.